### ISSUE TITLE
Multiplatform Fork

### ISSUE THREAD

**Issue by maikramer** [BoundingBoxSoftware/Materialize#6]
I worked all day to port Materialize to be multiplatform, the repository is in https://github.com/maikramer/Materialize
Tested only in linux
Available in release, linux only.

Update, it's working great in linux now

**Comment by fouman75** [comment]
Cool! I made a fork as well for OSX: https://github.com/fouman75/Materialize - osx branch
I'll check out your version since I removed a lot of code. Maybe your multiplatform version is the way to go.

**Comment by rszibele** [comment]
Was just checking out Materialize and was sad to find out there was no Linux port available, until I found this issue.

It would be awesome if these ports get listed on the official page for new users as one has to dig a bit to find them. I can't speak for the OS X fork as I haven't tried it, but the Linux port from @maikramer works flawlessly, thanks guys!

**Comment by maikramer** [comment]
> Was just checking out Materialize and was sad to find out there was no Linux port available, until I found this issue.
> 
> It would be awesome if these ports get listed on the official page for new users as one has to dig a bit to find them. I can't speak for the OS X fork as I haven't tried it, but the Linux port from @maikramer works flawlessly, thanks guys!

Thanks guy, my first port ever, you dont know how i'm glad to see that i'm helping somebody.

**Comment by maikramer** [comment]
New Version, really fast now, no more ImageSharp and with a native file picker, it's much more stable now, at least in linux.

https://github.com/maikramer/Materialize/releases

**Comment by zicklag** [comment]
This is great! It ran on my Ubuntu 16.04 laptop even with my weak embedded graphics card. Now I get to do some experimenting with it. :slightly_smiling_face: 

**Edit:** It works great! Thanks a lot @maikramer! This will help me and my team very much.

**Comment by owlboy** [comment]
@fouman75 The macOS build does not display a preview for me. The background/preview area is just black.

10.14.4 - AMD Radeon R9 M395X 4 GB

**Comment by fouman75** [comment]
@owlboy I don't think this repo is being maintained. I made the macport for fun but I'm not really using the app or supporting it.
 
Here is what I suggest:
- Create a new issue here: [https://github.com/fouman75/Materialize/issues](https://github.com/fouman75/Materialize/issues) so we at least get something documented outside this thread.
- Try to document as best you can, maybe with screenshots of what you see vs what you expect. From your one liner, I don't understand what you are referring to.
- I didn't upgrade my Mac to 10.14 yet so if the problem is related to that, not much I can do right now.
- @maikramer seems to be making a lot of changes supporting Linux. It might be a good idea to just take the branch and build it for Mac. Building is really simple. You just need to install UNITY and build it. You can try that with either my branch or @maikramer's.

Cheers!

**Comment by maikramer** [comment]
@owlboy
I'm maintaing and working almost everyday on it, i remade 90% of the program, vulkan, HDRP, Compute shaders, everything is new. But the big remake is in another branch, a beta one.
I can start to build a mac version, but i need a dedicated tester, if anyone want to be a candidate, i can easily do it. I can even build one tonight and send to the candidate.
I will build only when somebody send me a request to be the tester. The issue is this one.
https://github.com/maikramer/Materialize/issues/6

- I work 8 days and 12 i'm off, and now, i'm in my second working day, then maybe i won't work too much on the project.

**Comment by BenRacicot** [comment]
Hey @maikramer and @fouman75 have you guys had time to continue this for MacOS? What would you recommend to use at this point?

**Comment by fouman75** [comment]
Hey @BenRacicot, see @maikramer reply on Apr 6. I think this is your best bet at his point as I haven't touched anything since last year.

**Comment by maikramer** [comment]
Hi, i had no feedback for a mac release then i didnt work on it. But, i can see what i can do if anybody wants to test.

**Comment by BenRacicot** [comment]
@maikramer that's too bad! I'm sure people just don't realize how important your work on this has been. I would be willing to test it... My experience is limited in this space but I will test your work if you'd be willing to communicate here and there.

**Comment by maikramer** [comment]
@BenRacicot No problem, the real thing is that i dont have a mac for testing. But, i can try to build if you can even run and open the textures on it. The only thing i dont know if will work will be the file picker, but i can build and you test. Thanks.

**Comment by fouman75** [comment]
Keep me posted if needed. I can try and build on my Mac. I had changed the file browser in my version as well. Maybe we can merge code if needed. Maybe we should move the issue/discussion to @maikramer repo.
